### PR TITLE
[EDU-3689] refactor: update templates names in Doc

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue:

[EDU-3689]

### Changes

Updating documentation due to new names for three templates and new Fingerprint integration:
- Guides for each template.
- Guides page.
- Templates main reference.

### Additional links

https://aziontech.atlassian.net/browse/DEV-907

[EDU-3689]: https://aziontech.atlassian.net/browse/EDU-3689?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/guias/guides.mdx [hunk 1]
```diff
@@ -57,13 +57,13 @@ permalink: /documentacao/produtos/guias/
...
-- [Como implantar o template Suite Image Cache](/pt-br/documentacao/produtos/guias/suite-image-cache/)
-- [Como implantar o template Suite Static Cache](/pt-br/documentacao/produtos/guiassuite-static-cache/)
-- [Como implantar o template Upstash Rate Limiting através da Azion](/pt-br/documentacao/produtos/guias/upstash-rate-limiting/)
 - [Como implantar o Fauna Boilerplate](/pt-br/documentacao/produtos/guias/fauna-template/)
 - [Como implantar o MongoDB Atlas Boilerplate](/pt-br/documentacao/produtos/guias/mongodb-atlas/)
+- [Como implantar o template Edge Application Proxy](/pt-br/documentacao/produtos/guias/edge-application-proxy-template/)
+- [Como implantar o template Hello World](/pt-br/documentacao/produtos/guias/hello-world-template/)
+- [Como implantar o template Image Optimization](/pt-br/documentacao/produtos/guias/image-optimization-template/)
+- [Como implantar o template Static Cache](/pt-br/documentacao/produtos/guias/static-cache-template/)
+- [Como implantar o template Upstash Rate Limiting através da Azion](/pt-br/documentacao/produtos/guias/upstash-rate-limiting/)
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/guides.mdx:66]: é pra só alguns terem `através da Azion`, mesmo?

**Review by hannah-sk — APPROVED** [review]

File: src/content/docs/pt-br/pages/guias/guides.mdx (hunk 1 above)
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/guides.mdx:66]:
```suggestion
- [Como implantar o template Upstash Rate Limiting](/pt-br/documentacao/produtos/guias/upstash-rate-limiting/)
```

File: src/content/docs/pt-br/pages/guias/guides.mdx [hunk 2]
```diff
@@ -57,13 +57,13 @@ permalink: /documentacao/produtos/guias/
...
 - [Como migrar um site WordPress para o edge com WordPress EdgeAccelerator](/pt-br/documentacao/produtos/guias/wordpress-edgeaccelerator/)
 - [Como usar o template Edge Function GitHub AutoDeploy](/pt-br/documentacao/produtos/guias/edge-function-github-autodeploy/)
 - [Como usar o template Upstash GeoLocation EdgeDeploy através da Azion](/pt-br/documentacao/produtos/guias/upstash-geolocation-edgedeploy/)
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/guides.mdx:69]:
```suggestion
- [Como usar o template Upstash GeoLocation EdgeDeploy](/pt-br/documentacao/produtos/guias/upstash-geolocation-edgedeploy/)
```

File: src/content/docs/en/pages/guides/index.mdx [hunk 3]
```diff
@@ -56,12 +56,12 @@ permalink: /documentation/products/guides/
...
+- [How to deploy the Edge Application Proxy template](/en/documentation/products/guides/edge-application-proxy-template/)
 - [How to deploy the Fauna Boilerplate](/en/documentation/products/guides/fauna-template/)
 - [How to deploy the Hello World template](/en/documentation/products/guides/hello-world-template/)
+- [How to deploy the Image Optimization template](/en/documentation/products/guides/image-optimization-template/)
 - [How to deploy the MongoDB Atlas Boilerplate](/en/documentation/products/guides/mongodb-atlas/)
-- [How to deploy the Suite Bypass template](/en/documentation/products/guides/suite-bypass/)
-- [How to deploy the Suite Image Cache template](/en/documentation/products/guides/suite-image-cache/)
-- [How to deploy the Suite Static Cache template](/en/documentation/products/guides/suite-static-cache/)
+- [How to deploy the Static Cache template](/en/documentation/products/guides/static-cache-template/)
 - [How to deploy the Upstash Rate Limiting template through Azion](/en/documentation/products/guides/upstash-rate-limiting/)
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/index.mdx:65]:
```suggestion
- [How to deploy the Upstash Rate Limiting template](/en/documentation/products/guides/upstash-rate-limiting/)
```

**Review by bru-andrade — APPROVED** [review]

**Pull request merged**